EasyPost.Tests/ShipmentTest.cs

```diff
@@ -14,11 +15,11 @@ public void Initialize()
         }
 
         [TestMethod]
-        public void TestCreate()
+        public async Task TestCreate()
         {
-            VCR.Replay("create");
+            VCR.Record("create");
```

Comment:
Why was this changed from `Replay` to `Record. I don't have much knowledge on scotch VCR yet, but this change seems inconsistent with other unit tests.

Reply:
Resolved in [13dec76](https://github.com/EasyPost/easypost-csharp/pull/235/commits/13dec7625f11be9e9d01d625178bc7e0fd723410)